docs/dev_tools/build_profiles.md

```diff
@@ -0,0 +1,21 @@
+# Build Profiles
+mbed 5.0 supports three primary build profiles, *default*, *debug* and *small*. The Online 
```

Comment:
mbed OS 5

Not 5.0 unless it's limited to that version, and not just "mbed" which is everything we do, including mbed Cloud.